quarkus-workshop-super-heroes/docs/src/docs/asciidoc/5a-contract-testing/happy-path-consumer-tests.adoc

```diff
@@ -0,0 +1,166 @@
+
+
+== The first contract test
+
+icon:hand-point-right[role="red",size=2x] [red big]#Call to action#
+
+Add the Quarkus extension to the `pom.xml` of the Fights service:
+
+[source,xml]
+----
+        <dependency>
+            <groupId>io.quarkiverse.pact</groupId>
+            <artifactId>quarkus-pact-consumer</artifactId>
+            <version>{quarkus-pact-version}</version>
```

Comment:
Weirdly, adoc variables don't have a $! (The idea is that the version is inlined in the adoc, not treated as a variable in the pom.)

Looks wrong, is right. :/ 

(Which is so annoying when reading it un-rendered.)

Reply:
Something is still broken with the build then because `{quarkus-pact-version}` should be replaced and it isn't...

![image](https://github.com/quarkusio/quarkus-workshops/assets/363447/0d79e8b7-b625-43f8-9424-5fd8cff90b12)